databroker/core.py

```diff
@@ -60,35 +62,46 @@ def fill_event(fs, event, handler_registry=None, handler_overrides=None):
                     event.data[data_key] = _fs.get_datum(value)
 
 
-class Header(doc.Document):
+@attr.s(frozen=True)
+class Header(object):
     """A dictionary-like object summarizing metadata for a run."""
 
+    _name = 'header'
```

Comment:
I gather that this is here for back-compat with the other doct variant. Can you confirm that its name was lowercase 'header'? I remember it being 'Header'.